Unity/Champloo/Assets/Scripts/Player/Movement States/Movement Specials/OnInfiniteDash.cs

```diff
@@ -0,0 +1,53 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class OnInfiniteDash : OnDash
+{
+    public override MovementState DecideNextState(Vector3 velocity, Vector3 externalForces)
+    {
+        specialTimeLeft -= Time.deltaTime;
+
+        //Buffer for attacks and movement specials done too early
+        if (specialTimeLeft < (specialTime * nextDashBufferWindow))
+        {
+            if (player.InputPlayer.GetButtonDown("Attack"))
+            {
+                earlyAttackInput = true;
+            }
+            else if (player.InputPlayer.GetButtonDown("Movement Special"))
```

Comment:
What is the expected behavior for a player spamming the dash button while they're in dash?